Extension/package.json

```diff
@@ -4561,7 +4561,7 @@
     "plist": "^3.0.5",
     "tmp": "^0.2.1",
     "vscode-cpptools": "^6.1.0",
-    "vscode-extension-telemetry": "^0.4.4",
+    "vscode-extension-telemetry": "^0.6.2",
```

Comment:
I thought it changed to `@vscode/extension-telemetry` for 0.6.2

Reply:
It did. I forgot to change the package name :D